dockerfiles/reverse-proxy/docker-compose.test.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+services:
+    revpro:
+        build: .
+        ports:
+            - "50132:80"
+        links:
+            - web
+        environment:
+            UPSTREAM_NAME: web
+            UPSTREAM_PORT: 80
+            FORCE_SSL: "false"
+            DISABLE_PROXY_PROTOCOL: "true"
+            FILTER_LOGS: "true"
+    web:
+        image: yeasy/simple-web
```

Comment:
[yeasy/simple-web](https://hub.docker.com/r/yeasy/simple-web/) is a small web server that just keeps a log of requests that go to it. I was looking for the simplest web server I could find to avoid having to run any config, since we're really just trying to test how the nginx reverse proxy works, and not the website it's in front of.

For those interested, the webpage looks like this:
![image](https://user-images.githubusercontent.com/12392541/87903797-33271980-caa0-11ea-9a3f-2cb376cf5d11.png)